core/modules/file/file.theme.inc

```diff
@@ -330,8 +330,8 @@ function theme_file_formatter_table($variables) {
       format_size($item['filesize']),
     );
   }
-  count($rows) > 1 ? $title = t('Attachments') : $title = t('Attachment');
-  $header = array($title, t('Size'));
+  $title = format_plural(count($rows), 'Attachment', 'Attachments');
+  $header = array(t($title), t('Size'));
```

Comment:
With `format_plural()`, `t()` isn't used. See the code used by [`admin_bar_links_alert()`](https://docs.backdropcms.org/api/backdrop/core%21modules%21admin_bar%21admin_bar.inc/function/admin_bar_links_alert/1), for example.

Reply:
Good catch @kiamlaluno 👍🏼 ...I gotta remember this.

Reply:
```suggestion
  $header = array($title, t('Size'));
```